_submissions/round-12/1/2015-02-22-plotting-function-R.md

```diff
@@ -0,0 +1,17 @@
+---
+date: 2015-02-22
+round: Round 12
+title: Second Meeting for Round 12
```

Comment:
Please change this to be the title for your blog post - it's automatically used the body of the generated page.